### PR TITLE
Issue #780 begun - Added manifest.json to OpenFirstLink Directory

### LINKED ISSUE #780: A chrome extension which opens the first link on the site in a new tab

### PR DESCRIPTION
**Pull request by ShreyasheeS**
This pull request fixes #780 
Further details about the kind of extension is required.
I have assumed the directory to have an html, a javascript and a png file.

### CODE REVIEW

File: addition/add.cpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include<iostream>
```
Comment (t2013anurag) [inline addition/add.cpp]: can you move this file inside a directory named basic_mathematics

Comment (ShreyasheeS) [reply addition/add.cpp]: @t2013anurag Sure!

Comment (ShreyasheeS) [reply addition/add.cpp]: @t2013anurag Should I create a new directory?

Comment (t2013anurag) [reply addition/add.cpp]: https://github.com/hacktoberfest17/programming/tree/master/mathematical_operations

I just added a commit which created this directory, move the file in this.

**Pull request closed without merging**